src/NuGet.Core/NuGet.Configuration/PackageNamespaces/PackageNamespacesConfiguration.cs

```diff
@@ -38,7 +38,7 @@ public IReadOnlyList<string> GetConfiguredPackageSources(string term)
             return SearchTree.Value?.GetConfiguredPackageSources(term);
         }
 
-        internal PackageNamespacesConfiguration(Dictionary<string, IReadOnlyList<string>> namespaces)
+        public PackageNamespacesConfiguration(Dictionary<string, IReadOnlyList<string>> namespaces)
```

Comment:
Are we making this public for test purposes?

I think you can use internalsvisibleto. 

I know in theory, it's not an ideal usage of the metadata, but that ship has long sailed, unless we aggressively make breaking changes.

Reply:
Ok. Addressed.